string color	#farbe
tb_base/SimplePosition position #die aktuelle Position (gerastert) und Ausrichtung